src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/routing.yml

```
@@ -63,6 +63,9 @@ akeneo_connectivity_connection_connect_marketplace_profile:
 akeneo_connectivity_connection_connect_marketplace_test_app_create:
     path: '/connect/marketplace/test-app/create'
 
+akeneo_connectivity_connection_connect_marketplace_test_app_delete:
+    path: '/connect/marketplace/test-app/{testAppId}/delete'
```

```suggestion
    path: '/connect/marketplace/test-apps/{testAppId}/delete'
    requirements:
        testAppId: '[a-zA-Z0-9-]+'
```
not sure if the `testAppId` is a uuid

---

it would be inconsistent  with `akeneo_connectivity_connection_connect_marketplace_test_app_create` declared above

---

Indeed so I think we should fix it too :-)

---

In a REST world, it should be `POST '/connect/marketplace/test-apps` for create endpoint

---

@masacc, those are not REST urls, those are urls for the front.
`/connect/marketplace/test-app/create` is not a `POST` url, it's the url opening the modal to create it.

The REST endpoint is defined in `src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/routing/marketplace.yml` like this:

```
akeneo_connectivity_connection_marketplace_rest_test_apps_create:
    path: '/test-apps'
    controller: Akeneo\Connectivity\Connection\Infrastructure\Marketplace\TestApps\Controller\InternalApi\CreateTestAppsAction
    methods: [POST]
```

Also, ok for renaming both routes from `/connect/marketplace/test-app/` to `/connect/marketplace/test-apps/`.